specification/src/main/asciidoc/platform/ResourcesNamingInjection.adoc

```diff
@@ -5159,5 +4471,3 @@ Containers may optimize the steps above,
 e.g., by avoiding calls to the actual CDI SPI and relying on
 container-specific interfaces instead, as long as the outcome is the
 same.
-
-'''
```

Comment:
Do not remove this ''' for generating a line...  This provides a break between the text and the footnotes in this section when generating the pdf.  I guess I should have put a comment indicating the need for this.  Could you just add something along the lines of // generates a line between text and footnotes for pdf and html generation.

Could you also add a similar comment line at the end of ApplicationProgrammingInterface (since you are updating that file as well)?

Thanks!